src/swagger/payment_detail/post.yml

```diff
@@ -0,0 +1,100 @@
+title: Create a new payment detail
+description: Create a new payment detail
+tags:
+  - Product-Detail
+operationId: postpaymentdetail
+
+requestBody:
+  description: Create a new payment detail
+  content:
+    application/json:
+      schema:
+        type: object
+        properties:
+          amount:
+            type: float
+          status:
+            type: string
+          orders_id:
+            type: integer
+          payment_methods_id:
+            type: integer

```

Comment:
Include an example below the type e.g

`
properties:

    amount:

        type: float

        example: 50,000.00

`